app/assets.py

```diff
@@ -30,3 +30,23 @@
     filters='jsmin',
     output='scripts/vendor.js'
 )
+
+asylum_scss = Bundle(
+    'asylum/main.css',
```

Comment:
Thoughts on moving forward?

Basically I have `asylum_scss` separate from `app_css` for a reason, but I'm not sure if the priority should be on fixing the `asylum_scss` bundle so it actually uses the `.scss` files or if we should just stick with the compiles `app.css` for now, and fix that later.